doc/user_guide/user_guide.md

```diff
@@ -34,14 +34,14 @@ eval $(luarocks path)
 Then install `exasol-driver-lua` and required [Lua libraries](../../dependencies.md) by running
 
 ```sh
-luarocks install --local luasqlexasol
+luarocks install --local luasql-exasol
```

Comment:
I assume it's correct, but on all other places it's 
`luasql.exasol`

So just wanted to mention it...

Reply:
Thanks! It's OK. We need `luasql.exasol` for loading modules and `luasql-exasol` will be the luarocks package name (it's not published yet). This follows the luasql naming conventions: https://keplerproject.github.io/luasql/